SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialVirtualWorkerTranslator.cpp

```diff
@@ -95,6 +95,8 @@ void SpatialVirtualWorkerTranslator::ApplyMappingFromSchema(Schema_Object* Objec
 	int32 TranslationCount = (int32)Schema_GetObjectCount(Object, SpatialConstants::VIRTUAL_WORKER_TRANSLATION_MAPPING_ID);
 	VirtualToPhysicalWorkerMapping.Reserve(TranslationCount);
 
+	UE_LOG(LogSpatialVirtualWorkerTranslator, Verbose, TEXT("(%d) Apply valid mapping from schema"), LocalVirtualWorkerId);
```

Comment:
```suggestion
	UE_LOG(LogSpatialVirtualWorkerTranslator, Verbose, TEXT("Completed virtual worker translation mapping received on virtual worker %d"), LocalVirtualWorkerId);
```

Reply:
Reworded, and should be a Log level I think, should only get run once per server

Reply:
So the idea it that I left the verbose logs for individual assignment, and added that line to distinguish them from the following ones that will report all the assignment when it is complete.

Reply:
Ah, my bad, thought this ran only once a successful mapping was done.

Reply:
Suggestion for reworded text still applicable